common/app/views/fragments/items/elements/facia_cards/image.scala.html

```diff
@@ -1,11 +1,7 @@
+@import conf.switches.Switches
 @import layout.WidthsByBreakpoint
 @import model.ImageMedia
-@import views.support.{ImgSrc, RenderClasses, SrcSet}
-@import implicits.Requests._
-@import conf.switches.Switches
-@import experiments.{ActiveExperiments}
```

Comment:
I presume you removed `implicits.Requests` & `experiments.{ActiveExperiments}` intentionally?

Reply:
I used IntelliJ's "Optimise Imports" - I think they weren't being used

Reply:
I'll add `implicits.Requests` back as I'm not 100% sure about that, but `ActiveExperiments` is no longer referenced in the file

Reply:
Sounds good!